airflow-core/src/airflow/ui/src/i18n/README.md

```diff
@@ -17,8 +17,188 @@
+
+When a conflict arises in a translation-related PR, the following steps will be taken in order:
+
+- The involved parties should first try to reach a consensus through discussion in the PR.
+- If no consensus is reached, a translation owner may decide the outcome.
+- If multiple translation owners are involved and cannot reach consensus, the code owner will decide. If the code owner is sponsored,
+they should base their decision on a neutral source (e.g., a third-party opinion, translation tool, or LLM such as ChatGPT or Claude).
+- If the conflict is between code owners, a PMC member will be involved to resolve the conflict.
+
+### 6.4. Relinquishing translation/code ownership
+
+- When a translation owner asks to relinquish their role, or becomes inactive, and there are no other translation owners available, the code owner should follow these steps:
+  - They should raise a PR for removal of the translation owner from the `.github/CODEOWNERS` file.
+  - They should post a thread in the dev. list that the translation owner relinquishes their role, and that they are looking for a replacement.
+  - If a replacement is found within a reasonable time, they should be approved according to the relevant steps in section 6.1 - and afterward the code owner should update the `.github/CODEOWNERS` file to reflect the new translation owner.
+  - Otherwise, the code owner should raise a vote in the dev. list for the removal if the translation from the codebase.
+- When a translation sponsor asks to relinquish their role, and there is at least one translation owner - they should ask in the dev. list for assigning another sponsor instead.
+
+
+## 7. Standards & guidelines
+
+- Translations should be based on the default language (English). When translating a language that has already a similar translation supported
+(e.g., Portuguese vs. Spanish), the other language might be used as a reference, but still the default language (English) should be the primary source for translations.
+- Translations should be accurate, maintaining original meaning and intent.
+- Translations should be complete, covering all terms and phrases in the default language.
+- Translation of technical terminology should be consistent (for example: Dag, Task, Operator, etc.).
+- Language should be polite and neutral in tone.
+- Usage language that defies Apache Airflow's [code of conduct](http://airflow.apache.org/code-of-conduct/) is prohibited.
+
+> [!NOTE]
+> In case that local conventions requires deviation from the following guidelines, exceptions may be requested via PR or the dev list.
+
+
+- Local conventions should be considered (e.g., date formats, number formatting, formal vs. informal tone, etc.)
+- Formatting, placeholders, and variable substitutions must be preserved.
+
+> [!WARNING]
+> In languages with different word order than English, or in Right-To-Left (RTL) languages, it is important to validate that the changes are properly reflected in the UI.
```

Comment:
1. What to do if they're not? Add/remove abbreviations? Take artistic freedom?
2. Should we require screenshots in PRs?
3. "languages with different word order than English" - there are too many languages that fit this description. I'm particularly worried about strings that mix multiple languages (and directions), e.g."ה-DAGים", which it is extremely easy to get wrong.